docs/reference/configuration/service-mesh/egress.md

```diff
@@ -25,7 +25,7 @@ overrides:
           value: 2
 ```
 
-See the values.yaml for additional details and configuration options.
+See the [values.yaml](../../../../src/istio/charts/uds-istio-egress-config/values.yaml) for additional details and configuration options.
```

Comment:
With the docs site this will not link as expected, so it should just be a direct github link on the main branch (will be broken initially, but work once merged).